portfolio/src/main/java/com/google/sps/servlets/ReplyTreeChartServlet.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
```

Comment:
Oops I guess I overlooked it until now, but I think you should change the files you created to Copyright 2020 (although I think it probably doesn't matter)